sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-decisions.dmn

```diff
@@ -240,6 +240,8 @@
       <output id="OutputClause_0f80yob" label="limitAwardBCAGIncomeCap" name="limitAwardBCAGIncomeCap" typeRef="string" />
       <output id="OutputClause_03ngb7h" label="limitAwardBCAGSlope" name="limitAwardBCAGSlope" typeRef="string" biodi:width="192" />
       <output id="OutputClause_0etja44" label="limitAwardBCAGThresholdIncome" name="limitAwardBCAGThresholdIncome" typeRef="string" />
+      <output id="OutputClause_040aoed" name="limitAwardCSLPThresholdIncome" typeRef="string" />
+      <output id="OutputClause_1ehlydt" label="limitAwardSBSDThresholdIncome" name="limitAwardSBSDThresholdIncome" typeRef="string" biodi:width="192" />
```

Comment:
Thanks for adding the name, which was not given in the version of DMN given in the story,